sdk/tests/conformance2/reading/read-pixels-from-fbo-test.html

```diff
@@ -369,9 +462,12 @@
     case gl.INT:
       return 'INT';
     case gl.UNSIGNED_SHORT_5_6_5:
-    case gl.UNSIGNED_SHORT_4_4_4_4:
+      return 'UNSIGNED_SHORT_5_6_5';
     case gl.UNSIGNED_SHORT_5_5_5_1:
+      return 'UNSIGNED_SHORT_5_5_5_1';
     case gl.UNSIGNED_INT_2_10_10_10_REV:
+      return 'UNSIGNED_INT_2_10_10_10_REV';
+    case gl.UNSIGNED_SHORT_4_4_4_4:
```

Comment:
It seems you already handled UNSIGNED_SHORT_4_4_4_4, so why drop the support here?